lib-utilities/config/config.go

```diff
@@ -151,6 +152,13 @@ type ConnectionMethodConf struct {
 	ConnectionMethodVariant string `json:"ConnectionMethodVariant"`
 }
 
+// EventConf stores all inforamtion related to event delivery configurations
+type EventConf struct {
+	DeliveryRetryAttempts        int `json:"DeliveryRetryAttempts"`        // holds value of retrying event posting to destination
+	DeliveryRetryIntervalSeconds int `json:"DeliveryRetryIntervalSeconds"` // holds value of retrying events posting in interval
+	RetentionEventsInMinutes     int `json:"RetentionEventsInMinutes"`     // holds value of how long we can retain the events
```

Comment:
Rename it to `RetentionOfUndeliveredEventsInMinutes`

Reply:
changed